### PR TITLE
Add initial template for contributing and issue template

### PR DESCRIPTION
**Pull request by aaabramov**
Initial drop for [issues/24](https://github.com/therajanmaurya/Stock-Hawk/issues/24).

Because I am not vary familiar with this project there are some thing to improve.  
For example:

- Should we add a placeholder for logs?
- Should reporter specify any additional environment details?

### CODE REVIEW

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+### Are you looking for help?
```
Comment (therajanmaurya) [inline .github/ISSUE_TEMPLATE.md]: Issue template looks good. I think we should add some others question too. like `summary` of the issue that would be first question and other questions, order looks fine. 

if you want to see an example checkout this https://github.com/therajanmaurya/android-client-2.0/blob/development/.github/ISSUE_TEMPLATE.md

File: .github/CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
+# How to become a contributor and submit your own code
+
+## Contributing A Patch
```
Comment (therajanmaurya) [inline .github/CONTRIBUTING.md:3]: It looks good

**Review by therajanmaurya — CHANGES_REQUESTED** [review]

**Comment by therajanmaurya** [comment]
@Abrasha please add PULL_REQUEST template too, see the example here https://github.com/therajanmaurya/android-client-2.0/blob/development/.github/PULL_REQUEST_TEMPLATE.md

**Comment by aaabramov** [comment]
@therajanmaurya done. Tell me if you would like these 4 commits to be squashed.

**Comment by therajanmaurya** [comment]
Yes please do

**Review by therajanmaurya — APPROVED** [review]

**Comment by aaabramov** [comment]
@therajanmaurya done!

**Comment by therajanmaurya** [comment]
Thanks @Abrasha

**Pull request merged**